open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-server/pom.xml

```diff
@@ -27,6 +27,13 @@
     </properties>
     <dependencies>
 
+
+        <dependency>
+            <groupId>org.quartz-scheduler</groupId>
+            <artifactId>quartz</artifactId>
+            <version>2.2.1</version>
```

Comment:
Also the group/artifact/version should be added into the '<dependencyManagement>' section in the top level pom.xml, then here we remove the version. This helps to maintain consistency and is documented in the developer guidelines